controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -284,11 +289,13 @@ function execute($request) {
 
 			if ($issue) {
 
+				// set publication date
+				$submission->setDatePublished($this->getData('datePublished') ? $this->getData('datePublished') : Core::getCurrentDate());
```

Comment:
Here my first draft of a technical description: https://pkp.sfu.ca/wiki/index.php?title=Versioning

> And, as I said, I am a little bit worried about the case above: if I first assigned the article to a future issue and then to a real issue, which publication date will this article (in the version 1) have...

I tested it: the date remains the same. But I am not sure if we can/should prevent that ... at least the user can always change the date by hand ... 🤔